pack-hdf-converters.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Set shell attributes so that way the script exits early on error, etc.  See the code block in "Writing robust scripts and debugging" in https://bertvv.github.io/cheat-sheets/Bash.html

Do bat files have similar checks?  If so, enable them.